### ISSUE TITLE
How do I set the launch parameter "-tickrate 100"

### ISSUE THREAD

**Issue by CoffeeRong** [Left4DevOps/l4d2-docker#11]
How do I set the launch parameter "-tickrate 100"？I want to optimize my server.
i don't know you mean 
"
Command line replacement
You can completely replace the guided environment variable configuration by providing arguments when starting the container
docker run --name l4d2-command-line \
left4devops/l4d2 +hostname "Vannah Hotel"
"

**Comment by CoffeeRong** [comment]
I see "DEFAULT-MODE coop" in the environment settings, does this mean that I can set "DEFAULT-TICKRATE 100"? I tried it, but it didn't work

**Comment by hazmeister** [comment]
I haven’t included a variable for tick rate as changing tick rate is not supported in vanilla l4d2, you need an addon . If you have the addon , set the tick rate using EXTRA_ARGS.

e.g `-e EXTRA_ARGS="-tickrate 100"`

if you’ve set it correctly, you should see it when you run `docker top l4d2`

**Comment by CoffeeRong** [comment]
Thank you. I added "EXTRA-ARGS" and "tickrate 100" to the "Environment" settings of Docker, and it seems to have been successfully implemented! This is my first time working on servers and Dockers. Thank you for your help and dedication!

**Comment by hazmeister** [comment]
No worries, you’re most welcome! 🤗

**Comment by CoffeeRong** [comment]
I have identified a new issue where the server encounters an error upon installing the **sourcemod** plugin, which is reported as 

> `./srcds_run: line 312: 64 Aborted (core dumped) $HL_CMD` 
 `Add "-debug" to the ./srcds_run command line to generate a debug.log to help with solving this problem`
`Server restart in 10 seconds`

After reinstalling and meticulously modifying the configuration, I have determined that the issue arose from the installation of the **sourcemod** plugin, rather than the "metamod" plugin. 

Have I overlooked a parameter that needs to be configured? I have uploaded the log at the end. 
I installed this using the Container Manager software on Synology.
[l4d2.csv](https://github.com/Left4DevOps/l4d2-docker/files/15321084/l4d2.csv)
![QQ截图20240515200027](https://github.com/Left4DevOps/l4d2-docker/assets/45674719/0ec48e72-04ad-4d47-850a-910f078975cd)
